test/knex.test.js

```diff
@@ -259,6 +260,77 @@ describe('transaction', function () {
 
     throw new Error('expected the call to throw')
   })
+
+
+  it('adopt-commit', async () => {
+
+    let foo1_id
+
+    const trx = await knex(DbConfigPG).transaction()
```

Comment:
after this line, use knex to update the database
we need to verify that we can work knex outside seneca
NOTE: 
```
// ... trx database updates
```

Reply:
Ok, done